ner/src/main/java/edu/illinois/cs/cogcomp/ner/Main.java

```diff
@@ -121,9 +124,9 @@ protected int processArgument(String[] args, int current) throws Exception {
             bracketedOutput = false;
         else {
             if (new File(args[current]).exists()) {
-                System.out.println("Loading properties from " + args[current]);
```

Comment:
looks like a command line app -- print to STDOUT (or STDERR if it's a warning etc)